modules/opengl/src/openglcapabilities.cpp

```diff
@@ -41,6 +41,8 @@
 
 namespace inviwo {
 
+// Or minimal opengl version is 3.3 and glsl version 330
```

Comment:
Our

Reply:
We required 3.3 before also... I just had it wrong in the readme...